code/Forms/UsedOnTable.php

```diff
@@ -72,6 +72,10 @@ public function usage(HTTPRequest $request)
         $record = $this->getRecord();
         $usage = $record->findOwners();
 
+        $usage = $usage->filterByCallback(function($item) {
+            return !$item->config()->get('hide_in_campaigns');
```

Comment:
this is nasty and we definitely shouldn't do it.  it's creating a coupling between campaigns and the used on table when there doesn't need to be one

the way to remove stuff from the used on table is via the updateUsage extension hook